x-pack/legacy/plugins/canvas/types/canvas.ts

```diff
@@ -51,3 +51,8 @@ export interface CanvasWorkpad {
   pages: CanvasPage[];
   width: number;
 }
+
+export type CanvasTemplate = CanvasWorkpad & {
+  help: string;
+  tags: string[];
+};
```

Comment:
When going over the template structure, the two differences I could spot were the help section and the tags. Let me know if I missed something though

Reply:
I think this works.  @cqliu1 could double-check me on this, though, as she implemented tags.